cv.tex

```diff
@@ -29,9 +33,11 @@
 
 \lastupdated % Print the Last Updated text at the top right
 
-\namesection{Debarghya}{Das}{ % Your name
-\urlstyle{same}\url{http://debarghyadas.com} \\ % Your website, LinkedIn profile or other web address
-\href{mailto:dd367@cornell.edu}{dd367@cornell.edu} | 607.379.5733 % Your contact information
+\namesection{Joe}{Pate}{ % Your name
+\urlstyle{same}\url{https://linkedin.com/in/whoisjoepate/} \\ % Your website, LinkedIn profile or other web address
+\href{mailto:stress@lotsofmoving.parts}{stress@lotsofmoving.parts} | 302.440.4540 \\% Your contact information
+\sectionspace % Some whitespace after the section
+I'm a decadal Information Security practitioner with focus on security incident detection and response. I enjoy making and integrating tools to improve operational process.
```

Comment:
(JC):
In your summary/objective, should “process” be plural? I’m torn.

Reply:
lets do it